### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by lfekson** [electron-userland/electron-builder#3222]
Hello,

my concern is only about lack of documentation. We use Ember and therefore run electron-builder indirectly on MacOS via "nsis" target for "win" platforms. Maybe because of that the initial files like app-update.yml, etc (which ones?) are not created. I've searched but didn't find anything about what exactly should these files include and where exactly they should be located. Is it possible to add some description to your documentation for auto-update configuring for a generic https server?

Thanks in advance.

**Comment by develar** [comment]
https://www.electron.build/auto-update#file-generated-and-uploaded-in-addition

> indirectly on MacOS via "nsis" target for "win" platforms

Do you mean prepackaged?

**Comment by lfekson** [comment]
No, just a usual command "ember electron:compile --platform win32 --arch x64 -prod".

**Comment by develar** [comment]
Ember uses electron-packager as far I know.

**Comment by lfekson** [comment]
Sorry, I didn't dig too deep into all the dependencies, but know that electron-forge is the main point. As electron-forge didn't support nsis until recently, we used "squirrel" target. Since electron-forge-maker-nsis appeared it's possible to use "nsis" target. But built-in electron autoUpdate doesn't work with nsis, so we had to search for another way to auto-update. I've installed electon-build and electron-updater as the documentation says and added ...checkAndNotify... When .exe-file runs, it really calls some update and fails because no "app-update.yml" file found. I'd like to try to create needed files manually for a start, but content of the files is not described.
